fiware-region-sanity-tests/sanity_checks

```diff
@@ -141,7 +144,14 @@ done
 }
 TESTS=${TESTS:-tests/regions}
 
-# Main
+# Delete all generated Nova Console-Log files for the given regions before each sanity check execution.
+echo "Deleting Nova Console-Log files from previous executions for the given regions (${REGION_NAME_LIST[@]})"
+for region in ${REGION_NAME_LIST[@]}; do
```

Comment:
Although this is valid, please avoid using arrays in the shell script. Instead, as simple space-separated string value may suffice.